File: frontend-api/src/main/java/no/nav/melosys/tjenester/gui/dto/inntekt/FrilansInntektInformasjonDto.java
package no.nav.melosys.tjenester.gui.dto.inntekt;

import java.util.List;

import no.nav.melosys.domain.dokument.inntekt.ArbeidsInntektInformasjon;
import no.nav.melosys.domain.dokument.inntekt.ArbeidsforholdFrilanser;

public class FrilansInntektInformasjonDto {
    public List<ArbeidsforholdFrilanser> arbeidsforholdFrilanserListe;

    public FrilansInntektInformasjonDto(ArbeidsInntektInformasjon arbeidsInntektInformasjon) {
        arbeidsforholdFrilanserListe = arbeidsInntektInformasjon.getMutableArbeidsforholdListe();
    }
}

File: frontend-api/src/main/kotlin/no/nav/melosys/tjenester/gui/dto/inntekt/FrilansInntektInformasjonDto.kt
package no.nav.melosys.tjenester.gui.dto.inntekt

import no.nav.melosys.domain.dokument.inntekt.ArbeidsInntektInformasjon
import no.nav.melosys.domain.dokument.inntekt.ArbeidsforholdFrilanser

class FrilansInntektInformasjonDto(arbeidsInntektInformasjon: ArbeidsInntektInformasjon) {
    var arbeidsforholdFrilanserListe: List<ArbeidsforholdFrilanser>? = arbeidsInntektInformasjon.arbeidsforholdListe
}
